pootle/apps/pootle_app/views/admin/__init__.py

```diff
@@ -7,6 +7,9 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
-from .languages import *
-from .projects import *
-from .users import *
+from .languages import LanguageAdminView, LanguageAPIView
+from .projects import ProjectAdminView, ProjectAPIView
+from .users import UserAdminView, UserAPIView
+
+(LanguageAdminView, LanguageAPIView, ProjectAdminView, ProjectAPIView,
+ UserAdminView, UserAPIView)
```

Comment:
> Does feel like a cludge to get tests to pass. I'm OK with that but can we at least be consistent in placement/use of tuple?

Yep it is a kludge but its kinda *the* kludge. We could use the __all__ var ie:

```
from foo import ClassForReexport, OtherClassForReexport


__all__ = [ClassForReexport, OtherClassForReexport]
```
but its really only a cosmetic difference